src/RestClient.cc

```diff
@@ -284,6 +284,12 @@ RestResponse Rest::Request(HttpMethod _method,
   // Set the default value: do not prove that SSL certificate is authentic
   curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, 0L);
 
+  // Follow redirects to any URL set on the Location header.
+  curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
+
+  // Set cURL to only follow 3 redirects tops.
+  curl_easy_setopt(curl, CURLOPT_MAXREDIRS, 3L);
```

Comment:
Do we want to do this for all HTTP requests, regardless of verb? This option will convert any `301`, `302` or `303` HTTP request to `GET`, regardless of verb.

I don't see an issue with this, but @nkoenig might have more insight on the matter. If we want to enable this for `GET` requests only, we should move it into the [GET method options block](https://github.com/gazebosim/gz-fuel-tools/pull/330/files#diff-c042ebcbcc7034d78f91c0b671efc9bf1347b5e97857700ba285f2cd948293aaR296).

Reply:
I considered that when first starting writing this down, and I don't any strong opinion. Redirecting can be implemented on `POST` verbs as well, so it makes sense to make this agnostic of the verb.